mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -171,6 +174,11 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_33262 onItemEntityDestroyed (Lnet/minecraft/class_1542;)V
 		ARG 1 entity
 	METHOD method_40133 streamTags ()Ljava/util/stream/Stream;
+	METHOD method_41406 itemMatches (Lnet/minecraft/class_6880;)Z
+		ARG 1 itemEntry
+	METHOD method_41407 itemMatches (Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_41409 getRegistryEntry ()Lnet/minecraft/class_6880;
```

Comment:
In `BlockState` we have `getBlockRegistryEntry`. Should this just be `getRegistryEntry` or `getItemRegistryEntry` for consistency?

Reply:
@liach I think you have the outdated version of Shnupbups' branch? That was renamed before the merge.